src/Middleware/EventDispatcherMiddleware.php

```diff
@@ -20,21 +20,16 @@ class EventDispatcherMiddleware implements MiddlewareInterface
     protected $eventDispatcher;
 
     /** @var array */
-    protected $events;
-
-    /** @var string */
-    protected $clientId;
+    protected $events = [];
 
     /**
      * Constructor
      *
      * @param string $clientId
      */
-    public function __construct(EventDispatcherInterface $eventDispatcher, $clientId)
+    public function __construct(EventDispatcherInterface $eventDispatcher, protected $clientId)
```

Comment:
**suggestion:** 
```suggestion
    public function __construct(protected EventDispatcherInterface $eventDispatcher, protected string $clientId)
```

Reply:
I've applied the 1st change but not the 2nd as it might be a BC break (could be done in a new PR I suppose, but I don't want to risk it now)